ospfd/ospf_ri.c

```diff
@@ -1192,7 +1192,11 @@ DEFUN (router_info,
   /* Check and get Area value if present */
   if (area)
     {
-      inet_aton (area, &OspfRI.area_id);
+      if (!inet_aton (area, &OspfRI.area_id))
+        {
+          vty_out (vty, "Please specifya valid Area ID%s", VTY_NEWLINE);
```

Comment:
typo ("specifya") :D
(and can we print the invalid value like in the other places?)